### PR TITLE
Handle side effects

### PR DESCRIPTION
**Pull request by jananga99**
[ITP-1178]

[ITP-1178]: https://fcodelabs.atlassian.net/browse/ITP-1178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <BrowserRouter>
+            <Routes>
```
Comment (udara-sinister) [inline src/App.tsx:10]: Use a separate file to handle routings

File: src/utils/firebaseConfig.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { initializeApp } from "firebase/app";
+import { getFirestore} from 'firebase/firestore';
+
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web firebaseConfig's Firebase configuration
+const firebaseConfigure = {
+    apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
```
Comment (udara-sinister) [inline src/utils/firebaseConfig.ts:11]: Use a .env file to store sensitive information

**Review by udara-sinister — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (jananga99) [reply src/App.tsx:10]: resolved in the Handling Issues(https://github.com/fcodelabs/training-web/pull/251) PR

File: src/utils/firebaseConfig.ts (hunk 2 above)
Comment (jananga99) [reply src/utils/firebaseConfig.ts:11]: Already using a separate .env file to store sensitive information and access them via process.env ?

**Pull request closed without merging**